etc/profile-a-l/clipgrab.profile

```diff
@@ -6,10 +6,14 @@ include clipgrab.local
 # Persistent global definitions
 include globals.local
 
+noblacklist ${HOME}/.config/ClipGrab
```

Comment:
Add a counterpart `blacklist ${HOME}/.config/ClipGrab` to disable-programs.inc so the noblacklist makes sense.